### PR TITLE
[WIP] Release build

### PR DESCRIPTION
**Pull request by sadovnikov**
This PR is for #113, functionality will be described in #113.

### CODE REVIEW

**Comment by remmelt** [comment]
@sadovnikov it would be nice if you could squash a bunch of these commits for easier review, or are you not in that stage yet?

File: build.gradle [hunk 1]
```diff
@@ -58,58 +52,60 @@ sonarRunner {
     }
 }
 
-def grgit = Grgit.open(project.file('.'))
-def tags = grgit.tag.list()
-def versions = []
+afterEvaluate {
+}
```
Comment (frankscholten) [inline build.gradle]: Is this empty `afterEvaluate` block still needed?

Comment (frankscholten) [inline build.gradle]: It's out of date with `master`

**Comment by frankscholten** [comment]
@remmelt Those commits are for testing the release process itself. I agree we should squash them and create a clean branch when we are ready to merge.

**Comment by alex-glv** [comment]
@sadovnikov would be nice to have a short summary of changes and the approach

File: build.gradle [hunk 2]
```diff
@@ -58,58 +52,60 @@ sonarRunner {
...
+    exec {
+        workingDir "${project.rootDir}"
+        commandLine "git", "push"
     }
 }
 
-Version currentVersion = new Version.Builder("0.0.0").build()
+task resetVersionInBashScript << {
+    // have to use regular expression because the project version has changed and the previous is not kept
+    ant.replaceregexp(file: "bin/minimesos", match: "MINIMESOS_TAG=\".*\"", replace: "MINIMESOS_TAG=\"latest\"")
```
Comment (alex-glv) [inline build.gradle]: I don't think it's likely to have multiple double quotes on one line, but I usually prefer non-greedy match
```.*?``` to terminate at the first match

**Comment by sadovnikov** [comment]
Thanks for your first reviews.

@frankscholten, @remmelt I would need your help with squashing the commits. I tried, but seems only messed up the graphs even more

@alex-glv Are you looking for something like https://github.com/ContainerSolutions/minimesos/issues/113#issuecomment-200425774 ?

**Comment by frankscholten** [comment]
@sadovnikov Create a separate branch based on this one and squash commits with `git rebase -i` https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

**Pull request closed without merging**